website/mod.ts

```diff
@@ -41,7 +41,7 @@ export interface Props {
   routes?: Routes[];
 
   /** @title Seo */
-  seo?: Omit<Seo, "jsonLDs">;
+  seo?: Omit<Seo, "jsonLDs" | "titleTemplate" | "descriptionTemplate" | "canonical">;
```

Comment:
why did you change this? it seems to be unrelated with the PR title

Reply:
@mcandeia I just rebased the pr with the current main and this come with that rabse automatically